src/components/Agent/AgentMiniList.vue

```diff
@@ -93,8 +93,7 @@ export default {
   },
   methods: {
     ...mapMutations('agent', ['updateSelectedAgents']),
-    ...mapMutations('target', ['removeTargetEntityToDelete']),
-    ...mapMutations('general', ['addEntityToDelete']),
+    ...mapMutations('general', ['addEntityToDelete', 'removeTargetEntityToDelete']),
```

Comment:
why do we have on Agent component to use this target method removeTargetEntityToDelete ??